Microsoft.Toolkit.Uwp.SampleApp/SamplePages/Facebook Service/FacebookCode.bind

```diff
@@ -23,7 +23,7 @@ await FacebookService.Instance.PostToFeedWithDialogAsync(TitleText.Text, Descrip
 await FacebookService.Instance.PostPictureToFeedAsync(TitleText.Text, picture.Name, stream);
 
 // Get current user's photo albums
-await FacebookService.Instance.GetUserAlbumsAsync();
+await FacebookService.Instance.GetUserAlbumsAsync(20, FacebookAlbum.Fields);
```

Comment:
methods with original signatures have been added.. should we not leave them as is ? or are we marking those obsolete and suggesting use of new ones ?